src/routes/profile/index.tsx

```diff
@@ -1,5 +1,5 @@
 import { h, Component } from 'preact';
-import Button from 'preact-material-components/Button';
+import Button from 'preact-material-components/Button/Button';
```

Comment:
Could just rename the `Button.d.ts` etc. to `index.d.ts` so that it works without modification.